src/test/java/org/fundacionjala/coding/kevincristhian/RentalTest.java

```diff
@@ -0,0 +1,45 @@
+package org.fundacionjala.coding.kevincristhian;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Rental test class.
+ */
+public class RentalTest {
+    private static final int DAYS_RENTED = 2;
+    private static final double DELTA = 0.1;
+    private static final double EXPECTED_COST = 8.0;
+    private static final int EXPECTED_POINTS = 3;
+
+    /**
+     * test of total cost.
+     */
+    @Test
+    public void calculateTotalCost() {
+        Movie movie;
+        Rental rental;
+        rental = new Rental();
```

Comment:
You are repeating this line of code in all your tests
Please move it to a @before method.